sdk-base/src/main/java/com/mapbox/maps/extension/observable/eventdata/CameraChangedEventData.kt

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.maps.extension.observable.eventdata
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * The data class for camera-changed event data in Observer
+ */
+data class CameraChangedEventData(
+  /**
+   * Representing timestamp taken at the time of an event creation, in microseconds, since the epoch.
+   */
+  @SerializedName("begin") val begin: Long,
+
+  /**
+   * For an interval events, an optional `end` property will be present that represents timestamp taken at the time
+   * of an event completion.
+   */
+  @SerializedName("end") val end: Long?
```

Comment:
why is it nullable? 🤔

Reply:
From Event class' documentation:
```
    /**
     * Generic container for an event's data (Object). By default, event data will contain `begin` key, whose value
     * is a number representing timestamp taken at the time of an event creation, in microseconds, since the epoch.
     * For an interval events, an optional `end` property will be present that represents timestamp taken at the time
     * of an event completion. Additional data properties are docummented by respective events.
     *
     * ``` text
     * Event data format (Object):
     * .
     * ├── begin - Number
     * └── end - optional Number
     * ```
     */
```